amida-tech/mcp-data-client
# Mcp 180 jf change uploadpage display

### Related tickets:
MCP-180

### What's changing? Any breaking changes?
Basically, this is the heart of the application. 

### Manual test cases?
1) Run `yarn test`. The tests at the moment cover all the front end components I've built. A different ticket will handle the DataService test that is required.
2) Start the service with `yarn start`. Ensure that `mcp-data-mapping-util` is running via `flask run`. If you've set the backend's port to be different, update `REACT_APP_MCP_DATA_MAPPING_UTIL` in `.env` in this project to reflect this. 
3) Upload a few files. Courtesy of @TylerNoblett, I've included basic files in `/sample-data` that will cover the seven basic error types as well as the properly validating test cases. 


### Any additional context/background?
Needs a Dockerfile, needs a test for Dataservice, and to correct a function name for Grant. Then we're probably good to go on delivery. 

### Screenshots (if appropriate):
-
